python/ray/train/mosaic/mosaic_trainer.py

```diff
@@ -207,14 +208,20 @@ def _mosaic_train_loop_per_worker(config):
     os.environ["WORLD_SIZE"] = str(session.get_world_size())
     os.environ["LOCAL_RANK"] = str(session.get_local_rank())
 
+    # Replace Composer's Loggers with RayLogger
+    config["loggers"] = RayLogger(keys=config.pop("log_keys", []))
+
     # initialize Composer trainer
     config["progress_bar"] = False
+    config["log_to_console"] = False
```

Comment:
can we do set those after the trainer is defined? it seems to me like the user can work around this if they just use different keys in config.

Reply:
updated -- we remove composer loggers from the reporting pipe (the callbacks) and also from destinations, to which the data will be relayed to (logger's destinations).

It is difficult to remove all the occurrences of each instance of composer loggers, since they are all passed in as arguments/attributes of state, Logger, Engine, etc. However, by removing them from the callbacks and logger destinations, we can suppress the logger behavior. (although the empty frame of the loggers may still exist)

Reply:
Thanks, that sounds good. Do we still need to modify the user-specified config dict by setting the `loggers` key?

Reply:
Good point -- updated so that instead of having the loggers key, the ray logger is added to the callbacks and logger destinations after trainer init.